src/main/java/frc/robot/commands/IntakeSet.java

```diff
@@ -0,0 +1,51 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+import frc.robot.subsystems.Intake;
+
+public class IntakeSet extends CommandBase {
+  /**
+   * Creates a new IntakeSet.
+   */
+  private Intake intake;
+
+  boolean setIntake; // True = On, False = Off
```

Comment:
This needs a better name, also change it in the constructor. My ideas right now are "state', "running", "isRunning", and "enabled",  but I'm pretty tired. We just need something that is a little more descriptive